client/document/index.jsx

```diff
@@ -45,8 +46,28 @@ class Document extends React.Component {
 			devDocs,
 			devDocsURL,
 			feedbackURL,
+			inlineScriptNonce,
+			analyticsScriptNonce,
 		} = this.props;
 
+		let inlineScript = `COMMIT_SHA = ${ jsonStringifyForHtml( commitSha ) };`;
```

Comment:
could we add `\n` at the end of those? I think it would slightly be more readable when viewing the source.